### ISSUE TITLE
Feature request: zoom rendered diagrams with mouse/touchpad

### ISSUE THREAD

**Issue by e13mort** [esteinberg/plantuml4idea#262]
It would be great to be able to zoom rendered diagrams. The behaviour might be the same as a text zoom gesture in IDEA (zoom rendered text then render it again with a new quality based on a new zoom level).

**Comment by krasa** [comment]
do you mean ctrl+wheel?

**Comment by e13mort** [comment]
Yes. I didn't know I could do that :) Anyway, there are few issues with this zooming: 

- It's laggy and works with perceptible delay
- Zoom doesn't work with touchpad

**Comment by krasa** [comment]
I just tried it on Dell with Win 10 and the trackpad works.

**Comment by krasa** [comment]
It did not work properly with OS scaling. Should be fixed now.

And it is laggy because plantuml png rendering is slow, implementing svg viewer could help a little bit.

**Comment by jpsenior** [comment]
This is a lovely feature, thank you for adding it! I only discovered it from the release notes.

**Comment by e13mort** [comment]
I think I forgot to mention that I have issues with touchpad zooming on MacOS

**Comment by krasa** [comment]
I could add some logging, or you could try to debug it yourself if you want.
